### ISSUE TITLE
No new Version of edited File

### ISSUE THREAD

**Issue by boyi01** [ONLYOFFICE/onlyoffice-nextcloud#292]
Hey,
i have NC18  with the integrated onlyoffice version, but when i edit a File no new Version is created...

**Comment by HelenBelova17** [comment]
Hello @**boyi01**,
please specify some additional information:
1. Version of the Document Server;
2. Version of the ONLYOFFICE integration app for Nextcloud;
3. Describe by steps your actions, please.

**Comment by boyi01** [comment]
Hey,
1. Community Document Service: 	0.1.5
2. ONLYOFFICE: 4.1.4
3.  I open a file xlsx in webbrowser --> Only Office starty
     I change something and save it
     I close the file
--> no new Version created but the file get saved

**Comment by HelenBelova17** [comment]
Please note that Community Document Server was developed by Nextcloud Team, if you have any issues with it, please contact them in the appropriate GitHub repository. 

If you are using our official method of integration (Nextcloud + dedicated Document Server instance as per our installation [guide](https://api.onlyoffice.com/editors/nextcloud)), we will be glad to answer your questions. Please see more info on our [forum](https://dev.onlyoffice.org/viewtopic.php?f=53&t=25444).

**Comment by boyi01** [comment]
I would say the the problem is starts when the only office safes the file back to nextcloud... and so it should be the connector an I'm here right

**Comment by Chartman123** [comment]
I can confirm this behaviour also with a dedicated Document Server, but I've seen it only once until now. I will add more information if it happens again.

Nextcloud 18.0.4
Document Server 5.5.3

**Comment by thmo** [comment]
Also seeing this problem with a separate Document Server.

Nextcloud 19.0.0 (in Docker)
Onlyoffice app 4.2.0
Document Server 5.5.3.39 (in Docker)

Even after 2h waiting, the document is not synced back, and I am not even sure how to force sync, or where to look for error messages.

**Comment by thmo** [comment]
FWIW, I see sporadic error messages in the log
```
[onlyoffice] Error: GuzzleHttp\Exception\RequestException: cURL error 1: Protocol "ws" not supported or disabled in libcurl
...
POST /apps/onlyoffice/track?doc=...
```
Not sure however whether that is related to the sync issue or not.

**Comment by LinneyS** [comment]
The initial question was about the Community Document Service

@Chartman123 @thmo 
you can create a new issue

**Comment by Chartman123** [comment]
> The initial question was about the Community Document Service
> 
> @Chartman123 @thmo 
> you can create a new issue

Haven't seen it anymore since the report :)
